include/zephyr/instrumentation/instrumentation.h

```diff
@@ -81,6 +81,12 @@ bool instr_tracing_supported(void);
  */
 bool instr_profiling_supported(void);
 
+/**
+ * @brief Checks if dynamic trigger configuration feature is available.
+ *
```

Comment:
```suggestion
 * @return ...
```

Reply:
Added `@return` here.

Btw. I've noticed that we're missing `@return`s for the rest of symbols as well. Would it be fine with you if I added those missing entries in a follow-up PR?